README.md

```diff
@@ -76,6 +82,15 @@ robot.switch_to_manipulation_mode()
 robot.set_ee_pose([0.5, 0.6, 0.5], [0, 0, 0, 1])
 ```
 
+### Development
+
+To develop in `home-robot`, install the git pre-commit hooks:
+```
+python -m pip install pre-commit
+cd $HOME_ROBOT_ROOT
+pre-commit install
+```
+
```

Comment:
This overlaps with the "Code Contribution" section in the same README, we should combine them.
Also this goes to show that maybe our top-level README is being clogged with too much information, maybe we should move some of them (like the contact-graspnet note) to the README in src/home_robot?

Reply:
I agree, let's file an issue to redo our documentation?
i think the top-level is pretty out of date, but agree we need a minimal setup

Reply:
https://github.com/facebookresearch/home-robot/issues/74